docs/types.rst

```diff
@@ -813,9 +813,14 @@ for each ``_KeyType``, recursively.
             balances[msg.sender] = newBalance;
         }
     }
+  
+::
+
+    pragma solidity ^0.4.0;
 
     contract MappingUser {
-        address contractAddress = 0x42;
+        // the address of a MappingExample contract
+        address contractAddress = 0x42...;
```

Comment:
Hi @axic,

Thanks for the review, does your message means that I should change `0x42...` back to `0x42`? or that actually the previous example works and do not need the proposed change?

Thanks!